source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -157,6 +181,48 @@ content: |
            npm package. See the `npmjs documentation on the uuid-base64
            package <https://www.npmjs.com/package/uuid-base64>`_ for
            installation instructions.
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+           :emphasize-lines: 14,15,24,30
+
+           from pymongo import MongoClient
+           from pymongo.encryption_options import AutoEncryptionOpts
+           from pymongo.encryption import ClientEncryption
+           import base64
+           from bson.codec_options import CodecOptions
+           from bson.binary import (Binary,
+                         JAVA_LEGACY,
+                         STANDARD,
+                         UUID_SUBTYPE)
+           OPTS = CodecOptions(uuid_representation=STANDARD)
+           connection_string = "mongodb://localhost:27017"
+           key_vault_namespace = "encryption.__keyvault"
+           fle_opts = AutoEncryptionOpts(
+               kms_providers,
+               key_vault_namespace,
+               mongocryptd_bypass_spawn = True 
+           )
+ 
+           client = MongoClient(
+               connection_string,
+               auto_encryption_opts = fle_opts 
+           )
+ 
+           client_encryption = ClientEncryption(
+               kms_providers,
+               key_vault_namespace,
+               client, 
+               OPTS 
+           )
+           data_key = client_encryption.create_data_key("local")
+ 
+           uuid_data_key_id = uuid.UUID(bytes=data_key)
```

Comment:
I noticed for optional params in function calls, sometimes you're using a space between the "=" and sometimes no space. Could you keep it consistent whether it's part of PEP8? I would recommend a single space on each side of the "=". There are other instances, so make sure that you check all the python snippets.

Reply:
Fixed:
* Ran code through python linter